src/content/docs/browser/browser-monitoring/tutorials/combining-npm-nerdgraph.mdx

```diff
@@ -0,0 +1,166 @@
+
+<img
+  title="Nerdgraph Explorer: Browser Application Config"
+  alt="Nerdgraph Explorer: Browser Application Config"
+  src={nerdgraphExplorerBrowserApplicationConfig}
+/>
+
+<Callout variant="tip">
+Need more information on using the NerdGraph APIs or Explorer? [Checkout the NerdGraph Introduction page](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/).
+</Callout>
+
+</Step>
+
+<Step>
+### Install the config
+
+Copy the contents of the `jsConfig` property and replace the contents of the `options` object your source file.
+</Step>
+</Steps>
+
+## Depoloying using the npm package
+
+With the npm package installed and properly configured, you can now build and deploy your browser application. Load your website in a browser and verify there are network calls being made to the `beacon` URI from the config.
+
+## Inject browser application config into HTML
+
+As an alternative to placing the browser application configuration in your source code, you may want to inject the configuration into the HTML of your website. This is especially useful when your source code is reused across multiple websites.
+
+<Steps>
+<Step>
+### Modify source code snippet
+
+Modify the first snippet in your source code to `import` and instantiate the `BrowserAgent` class with no parameters.
+
+```javascript
+import { BrowserAgent } from '@newrelic/browser-agent/loaders/browser-agent'
+// Remaining import statements
+
+// The agent loader code executes immediately on instantiation.
+new BrowserAgent()
+
+// Remaining code
+```
+</Step>
+
+<Step>
+### Modify the graphql query
+
+Modify the graphql query to retrieve the browser application config as a JSON encoded script string. Run the new query remembering to replace `<guid>` with the browser application entity guid retrieved earlier.
+
+```graphql
+query {
+  actor {
+    entity(guid: '<guid>') {
+      ... on BrowserApplicationEntity {
+        guid
+        name
+        browserProperties {
+          jsConfigScript
+        }
+      }
+    }
+  }
+}
+```
+</Step>
+
+<Step>
+### Install the config
+
+Copy the contents of the `jsConfigScript` property. You will need to run the results through a JSON parser or replace the values of `\"` with `"`. The result will be a string containing a `<script>` element with all the browser application config information. Place the `<script>` element at the top of the `<head>` element, but below any `<meta>` elements, of your website HTML.
+</Step>
+</Steps>
+
+## Taking it further
+
+This tutorial only scratches the surface of how you might use the NerdGraph APIs and the browser agent npm package. Some ideas for further integration:
+
+- Integrate the browser agent into your website builder application and use the NerdGraph APIs to retrieve the browser application config for each of your customers when they enable New Relic monitoring within your management dashboard.
```

Comment:
```suggestion
- Integrate the browser agent into your SaaS platform using the npm package and use the NerdGraph APIs to retrieve browser application configurations specific to each of your customers as they enable New Relic monitoring within your management dashboard.
```

Reply:
I've got nothing against website builders, but I thought maybe we could cast a broader net here.